sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -432,11 +433,6 @@ void stabilization_indi_init(void)
  */
 void stabilization_indi_enter(void)
 {
-  /* reset psi setpoint to current psi angle */
-  stab_att_sp_euler.psi = stabilization_attitude_get_heading_i();
```

Comment:
Why is this removed?

Reply:
This is meaningless here. `stab_att_sp_euler` is not the global variable used to set command, it is only needed to store the last command value (passed as a parameter of the run function) to send it later over telemetry.
If this reset is still needed (and maybe it is, @EwoudSmeur ?), it as to be done from upper level (guidance or rc_input).

Reply:
If when switching back to INDI (which we do e.g. when testing neural controllers) the heading does not jump then it is fine.